recipes/default.rb

```diff
@@ -17,6 +17,7 @@
 # limitations under the License.
 #
 
+node.default['openstack']['compute']['conf']['libvirt']['disk_cachemodes']='file=writeback,block=writeback'
```

Comment:
Please move this after line 51 where it uses a block that already contains the other attributes.